tests/get-criteria-test.js

```diff
@@ -55,12 +55,48 @@ describe('Criteria methods', function() {
         customTargetingKeyId: '597016'
       };
 
+      return expect(dfp.getCriteriaValueId(conditions))
+        .to.eventually.deep.equal('126109156816');
+    });
+
+  });
+
+  describe('getCriteriaValue', function() {
+
+    it('should return value id the same as getCriteriaValueId', function() {
+      var conditions = {
```

Comment:
optional: You could pull this `conditions` variable out above the test blocks since it looks like we reuse it a lot.